microsoft-edge/webview2/Roadmap.md

```diff
@@ -0,0 +1,54 @@
+---
+description: Learn what is coming next for WebView2
+title: Roadmap for Microsoft Edge WebView 2
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 05/19/2020
+ms.topic: reference
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, win32 apps, win32, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
+---
+
+# Microsoft Edge WebView2 Roadmap
+
+The WebView2 control that allows developers to embed web technologies in their native applications. This document outlines the prospective roadmap for WebView2 through 2020 and into 2021. WebView2 is under active development and the roadmap will continue to evolve based on market changes and customer feedback, so please note that the plans outlined here aren't exhaustive and are subject to change. 
+
+If you have concerns or questions about our Roadmap, please leave us feedback in our [feedback repo](https://github.com/MicrosoftEdge/WebViewFeedback).
+
+The WebView2 team has a few major efforts underway:
+
+1. WebView2 Runtime Installer (Q4 2020)
+2. General Availability 
+    - Win32 C/C++ (Q4 2020)
+    - .NET (Q4 2020)
+    - WinUI 3.0 (Q2 2021)
+3. Fixed Version (Q4 2020)
+4. Cross-Platform (TBD)
```

Comment:
I wouldn't call out cross-platform if the premise is "The WebView2 team has a few major efforts underway". The overall message is we are hearing xplat signals and actively listening to developer feedback and scenario, rather than committing supporting anything.